crates/uv/tests/common/mod.rs

```diff
@@ -321,7 +321,7 @@ impl TestContext {
     }
 
     /// Create a `uv add` command for the given requirements.
-    pub fn add(&self, reqs: &[&str]) -> std::process::Command {
+    pub fn add(&self, reqs: &[&str], dev: bool) -> std::process::Command {
```

Comment:
This isn't our usual pattern, I'd specify `.arg("--dev")` on each invocation instead.

Reply:
That's nicer, updated.